### PR TITLE
note shell is not taught

### PR DESCRIPTION
**Pull request by tefer0**
On line 17, and Unix shell between R and python programming, as part of what is not taught in programming.
#checkout_process

<details>
<summary><strong>Instructions</strong></summary>

Thanks for contributing! :heart:

If this contribution is for instructor training, please email the link to this contribution to
instructor.training@carpentries.org so we can record your progress. You've completed your contribution
step for instructor checkout by submitting this contribution!

Keep in mind that **lesson maintainers are volunteers** and it may take them some time to
respond to your contribution. Although not all contributions can be incorporated into the lesson
materials, we appreciate your time and effort to improve the curriculum. If you have any questions
about the lesson maintenance process or would like to volunteer your time as a contribution
reviewer, please contact The Carpentries Team at team@carpentries.org.

You may delete these instructions from your comment.

\- The Carpentries
</details>

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Carpentries Instructor Training has the following goals:
 * Prepare you to use these teaching skills in teaching [Carpentries workshops](https://carpentries.org/workshops/).
   
 Because we have only limited time, some things are beyond the scope of this training. We will not be learning:
-* How to program in R or Python, use Git or SQL, or any of the other topics taught in Data Carpentry, Library Carpentry, or Software Carpentry workshops.
+* How to program in R, Unix shell or Python, use Git or SQL, or any of the other topics taught in Data Carpentry, Library Carpentry, or Software Carpentry workshops.
```
Comment (brownsarahm) [inline index.md:17]:
```suggestion
* How to program in R or Python, use the Unix shell, Git or SQL, or any of the other topics taught in Data Carpentry, Library Carpentry, or Software Carpentry workshops.
```

**Review by brownsarahm — CHANGES_REQUESTED** [review]
The Unix shell is not a programming language, but it could be included in this sentence in a different place.

File: index.md (hunk 1 above)
Comment (tefer0) [reply index.md:17]: Thank you for the correction

**Review by tefer0 — COMMENTED** [review]
Thank you for the correction.

**Review by brownsarahm — APPROVED** [review]

**Pull request merged**